doc/examples/bundle_extraction.py

```diff
@@ -115,7 +115,28 @@
 model_af_l_file, model_cst_l_file = get_two_hcp842_bundles()
 
 """
-Extracting bundles using recobundles [Garyfallidis17]_
+Extracting bundles using RecoBundles [Garyfallidis17]_
+
+RecoBundles requires a model (reference) bundle and tries to extract similar
+looking bundle from the input tractogram. There are some key parameters that
+users can set as per requirements. Here are the key threshold parameters and
+their function in Recobundles:
+
+    - model_clust_thr : It will use QuickBundles to get the centroids of the
+    model bundle and work with centroids instead of all streamlines. This helps
+    in making RecoBundles faster. If you prefer to use all the streamnlines of
```

Comment:
I would add here that the larger the number is the fewer centroids there are and also explain what the units are -- this is mm? Voxel? Some kind of probability? What is the range of acceptable values?